### ISSUE TITLE
Support option --api for Stable Diffusion WebUI Forge (probably activating it by default)

### ISSUE THREAD

**Issue by cyberlytics** [LykosAI/StabilityMatrix#999]
### Proposal

Dear StabilityMatrix Developers, thank you for your great product!

I noticed that for Stable Diffusion WebUI Forge, the option --api is not available in the clickable/searchable package launch options. I consider it quite important, because it activates /sdapi/** REST-API of Forge (e.g., /sdapi/v1/txt2img).

(For the sake of completeness: When not providing --api the http://127.0.0.1:7860/docs/ still lists some REST-APIs, but the ones /sdapi/** interfaces are missing, which let you speak to the generators. I activated --api in StabilityMatrix as custom package launch option for Stable Diffusion WebUI Forge and got all relevant Web-APIs (http://127.0.0.1:7860/docs). If one know it, all works.)

I would now like to recommend, providing --api as clickable/searchable launch option, prominently just after host and port.

And you might consider, to apply it even by default. Allowing for local automatization without further ado.

This should apply to 1) Stable Diffusion WebUI Forge.
But also to 2) Stable Diffusion WebUI and 3) Stable Diffusion WebUI reForge.
(And might be applicable to 4) Stable Diffusion WebUI UI-UX, but I do not know it exactly.)

Kind regards
Christoph

**Comment by MrDohnny** [comment]
yeah , i've been having some problems with reForge and Auto Painter AI Blender Addon, because they try to make a request to /sdapi/v1/txt2img, when you use it in blender and nothing happens .
I think that i'll need to install StabilityMatrix to use it... so boring.
